helm/managed/logging/elastic/Chart.yaml

```diff
@@ -5,20 +5,23 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.1.0
+version: 1.0.0
+
+# Elastic Stack version
+appVersion: 7.12.0
```

Comment:
it probably doesn't matter, but where does 7.12.0 come from? should it be 8.2.2 
@richardkchapman - I'm not sure if the go*.sh scripts update version in examples - or if they should?

Reply:
I think 7.12 is the elk version rather than the HPCC version?

Reply:
And for that reason I'm not sure that the go*.sh scripts should change anything in managed subdir.

Reply:
Most of the charts published to our repo via the go. scripts report various chart/app versions already:
 helm search repo hpcc
NAME                    CHART VERSION   APP VERSION     DESCRIPTION
hpcc/elastic4hpcclogs   0.1.0                           A Helm chart for launching a lightweight Elasti...
hpcc/hpcc               8.0.24          8.0.24          A Helm chart for launching a HPCC cluster using...
hpcc/hpcc-azurefile     0.1.0           1.16.0          A helm chart to provision HPCC PVC's using the ...
hpcc/hpcc-efs           0.1.0           1.16.0          A helm chart to provision HPCC PVC's using the ...
hpcc/hpcc-filestore     0.1.0           0.1.0           A helm chart to provision HPCC PVC's using the ...
hpcc/hpcc-localfile     0.1.0           1.16.0          A helm chart to provision HPCC PVC's using the ...
hpcc/hpcc-localplanes   0.1.0           1.16.0          A helm chart to provision multiple HPCC PVC's u...
hpcc/hpcc-nfs           0.1.0           1.16.0          A Helm chart to provision HPCC PVC's using an N...
@richardkchapman